scripts/jenkins/cloud/ansible/roles/list_packages/vars/main.yml

```diff
@@ -0,0 +1,21 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+dirs_to_create_list:
```

Comment:
shouldn't you use those variables whenever you refer to `/tmp/rpms` ?

Why they are a list?

Reply:
legacy... In fact we can create /tmp/rpms/diffs from the begining(empty `diff` dir shouldn t be a problem) and then this file is not needed.